int32 id
float64 size
geometry_msgs/PoseStamped pose